src/hll.c

```diff
@@ -92,6 +112,318 @@ enum {
     MST_UNINIT		= 0xffff,	// Internal uninitialized.
 };
 
+// Support disabling hash aggregation functionality for PG > 9.6
+#if PG_VERSION_NUM >= 90600
+
+#define HLL_EXTENSION_NAME "hll"
```

Comment:
No need for HLL prefix, this is HLL extension anyway. We don't use HLL keyword anywhere in the codebase except functions exposed to user space.